Formula/xtrans.rb

```diff
@@ -28,6 +32,10 @@ class Xtrans < Formula
     depends_on "linuxbrew/xorg/xorg-sgml-doctools" => [:build, :recommended]
   end
 
+  if fedora?
+    patch :DATA
+  end
+
```

Comment:
We usually tend to use `inreplace` for small substitutions rather than `patch do`. Please try…
```ruby
inreplace "Xtranslcl.c", "# include <sys/stropts.h>", "# include <sys/ioctl.h>"
```